providers/dropboxsign/support.go

```diff
@@ -0,0 +1,24 @@
+package dropboxsign
+
+import "github.com/amp-labs/connectors/internal/datautils"
+
+//nolint:gochecknoglobals
+var (
+	objectNameTemplate     = "template"
+	objectNameBulkSendJobs = "bulk_send_job"
+	objectNameApiApp       = "api_app"
+	objectNameFax          = "fax"
+	objectNameFaxLine      = "fax_line"
+)
+
+//nolint:gochecknoglobals
+var readObjectResponseKey = datautils.NewDefaultMap(map[string]string{
+	objectNameTemplate:     "templates",
```

Comment:
could we fix the object display names when returning the metadata too? For example, `Api_app` looks weird in the UI. `API Apps` would be nice, same goes for `Fax lines`, etc